db/migrate/20220205090723_add_details_to_users.rb

```diff
@@ -0,0 +1,10 @@
+class AddDetailsToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :name, :string
+    add_column :users, :photo, :string
+    add_column :users, :bio, :string
```

Comment:
Please change the data type of the `bio` attribute to `text` as it will be holding a few lines of text but the `string` data type is for attributes such as `username`, and `password` etc.